src/BinaryParsers/PEBinary/ProgramDatabase/LinkerCommandLine.cs

```diff
@@ -100,5 +100,44 @@ public LinkerCommandLine(string commandLine)
                 this.IncrementalLinking = false;
             }
         }
+
+        public static bool IsLinkerCommandLine(string commandLine)
+        {
+            // The command line for link.exe should contain /OUT with the final binary name.  Use that to separate compiler and linker command lines.
+            // We should accept both / and - as the argument prefix.  The argument is case insensitive.  We do not know where in the command-line the
+            // OUT argument will be provided so we use Contains.
+            return ((commandLine != null) &&
+                    (commandLine.Contains("/OUT", System.StringComparison.OrdinalIgnoreCase) || commandLine.Contains("-OUT", System.StringComparison.OrdinalIgnoreCase)));
```

Comment:
>commandLine [](http://example.com/codeflow?start=21&length=11)

This code can call your new helper. :)

Reply:
I don't think so because it is `Contains`.  The /OUT option could appear anywhere in the long command-line string.  This is not called with one-by-one arguments like the other helpers.

Reply:
Right, yes, missed that point. We could have created CommandLineContains, I suppose, but with a single upstream caller, this factoring doesn't necessarily make sense for now.